scripts/ai-multi-agent.sh

```
@@ -0,0 +1,604 @@
+        # Marketer: 2人
+        print_info "起動中: marketer エージェント (2人)"
+        for i in {1..2}; do
+            start_agent_session "marketer" "$i"
+        done
+    elif [[ ${#@} -eq 2 ]]; then
+        # 特定のエージェントタイプと数を指定
+        agent_type=$1
+        count=$2
+        
+        # エージェントタイプの検証
+        if [[ ! " ${AGENT_TYPES[@]} " =~ " ${agent_type} " ]]; then
```

Shellcheck (0.10.0)

[error] Arrays implicitly concatenate in [[ ]]. Use a loop (or explicit * instead of @).

(SC2199)